style.css

```diff
@@ -80,3 +83,172 @@ nav #myLogo {
   font-size: 18px;
   color: #6070ff;
 }
+
+.grid-box {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #e5e5e5;
+  border-radius: 8px;
+  width: 100%;
+  gap: 88px;
+  padding: 104px 0;
+}
+
+.icon {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  padding: 0;
+}
+
+.box {
+  background-color: #fff;
+  gap: 10px;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  width: min(327px, 80vw);
+  border: 2px solid #dfe1e6;
+}
+
+.banner {
+  border-radius: 8px;
+  width: min(327px, 80vw);
+}
+
+input {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  background-color: #fff;
+  border-color: #6070ff;
+  border-radius: 8px;
+  padding: 8px 5px;
+  display: flex;
+  width: min(127px, 80vw);
+}
```

Comment:
- Good job with adjusting the height as spotted and suggested by previous reviewer 👍 It appears some parts of your project cards still need gentle touches. I have included some screenshots with annotations from your code and Figma design.  
- [ ] Please watch out for the following differences:
   - `font properties`
   - `color`
   - `margin`
   - `padding`
   - `height`, 
   - `border-color`
   - `button`, etc.
   
- **Your Design**

![image](https://user-images.githubusercontent.com/99715640/212441764-890e37f2-eb16-4c5f-8046-f76ceb739a21.png)

- **Figma Design**

![image](https://user-images.githubusercontent.com/99715640/212441870-576f3ae2-81f0-4674-b513-695a4bbd86d5.png)